clique-grpc/src/transport.rs

```diff
@@ -0,0 +1,50 @@
+use crate::{proto::client, server::GrpcServer};
+use clique::transport::{Request, Response, Transport};
+use futures::Future as Future01;
+use futures03::{
+    compat::Future01CompatExt,
+    future::{self, Ready},
+};
+use hyper::client::connect::HttpConnector;
+use std::future::Future;
+use std::net::SocketAddr;
+use std::pin::Pin;
+use tokio_sync::{mpsc, oneshot};
+
+pub struct GrpcTransport {
+    inner: client::MembershipService<tower_hyper::Client<HttpConnector, tower_grpc::BoxBody>>,
+    server: GrpcServer,
+}
+
+impl Transport<String> for GrpcTransport {
```

Comment:
Why not be generic over a `ToSocketAddr`?

Reply:
I think lazyness 😄